### PR TITLE
Remove observability tutorial and link to examples

### PR DESCRIPTION
**Pull request by chrkl**
**Description**

Changes proposed in this pull request:

- Remove observability tutorial and link to examples

**Related issue(s)**

https://github.com/kyma-project/kyma/issues/15914

### CODE REVIEW

**Comment by chrkl** [comment]
Merge after https://github.com/kyma-project/examples/pull/224

**Review by lindnerby — APPROVED** [review]

**Review by lindnerby — APPROVED** [review]

File: docs/03-tutorials/00-observability/README.md [hunk 1]
```diff
@@ -4,22 +4,22 @@ title: Observability - tutorials
...
 ## Sequence of tasks
 
 The instructions cover the following tasks:
 
  ![Monitoring tutorials](./assets/monitoring-tutorials.svg)
 
-1. [**Observe application metrics**](obsv-01-observe-application-metrics.md), in which you redirect the `cpu_temperature_celsius` metric to the localhost and the Prometheus UI. You later observe how the metric value changes in the predefined 10 seconds interval in which Prometheus scrapes the metric values from the service's `/metrics` endpoint.
-
-2. [**Create a Grafana dashboard**](obsv-02-create-and-configure-grafana-dashboard.md), in which you create a Grafana dashboard of a Gauge type for the `cpu_temperature_celsius` metric. This dashboard shows explicitly when the CPU temperature is equal to or higher than the predefined threshold of 75 degrees Celsius, at which point the dashboard turns red.
+1. [**Deploy a custom Prometheus stack**](https://github.com/kyma-project/examples/blob/main/prometheus/README.md), in which you deploy the [kube-prometheus](https://github.com/prometheus-operator/kube-prometheus) stack from the upstream Helm chart.
```
Comment (IwonaLanger) [inline docs/03-tutorials/00-observability/README.md:19]:
```suggestion
1. [**Deploy a custom Prometheus stack**](https://github.com/kyma-project/examples/blob/main/prometheus/README.md), in which you deploy the [kube-prometheus-stack](https://github.com/prometheus-operator/kube-prometheus) from the upstream Helm chart.
```

File: docs/03-tutorials/00-observability/README.md [hunk 2]
```diff
@@ -4,22 +4,22 @@ title: Observability - tutorials
 
 ## Purpose
 
-The following instructions describe the complete monitoring flow for your services in Kyma. You get the gist of Kyma built-in monitoring applications, such as Prometheus, Grafana, and Alertmanager. You will learn how and where you can observe and visualize your service metrics to monitor them for any alerting values.
+The following instructions describe the complete monitoring flow for your services in Kyma. You get the gist of monitoring applications, such as Prometheus, Grafana, and Alertmanager. You will learn how and where you can observe and visualize your service metrics to monitor them for any alerting values.
```
Comment (IwonaLanger) [inline docs/03-tutorials/00-observability/README.md:7]:
```suggestion
The following instructions describe the complete monitoring flow for your services in Kyma. You get the gist of monitoring applications, such as Prometheus, Grafana, and Alertmanager. You learn how and where you can observe and visualize your service metrics to monitor them for any alerting values.
```

Comment (IwonaLanger) [inline docs/03-tutorials/00-observability/README.md:7]: According to the content guidelines the present tense should be used rather than the future form.

**Review by IwonaLanger — APPROVED** [review]

**Pull request merged**